server/controllers/stock/reports/stock/consumption_graph.js

```diff
@@ -0,0 +1,73 @@
+const {
+  _, db, util, ReportManager, pdfOptions, STOCK_CONSUMPTION_GRAPTH_TEMPLATE,
+} = require('../common');
+const stockCore = require('../../core');
+const i18n = require('../../../../lib/helpers/translate');
+
+/**
+   * @method stockEntryReport
+   *
+   * @description
+   * This method builds the stock entry report as either a JSON, PDF, or HTML
+   * file to be sent to the client.
+   *
+   * GET /reports/stock/consumption_graph
+   */
+async function stockConsumptionGrathReport(req, res, next) {
+  try {
+
+    const params = _.clone(req.query);
+
+    const optionReport = _.extend(params, pdfOptions, {
+      filename : 'REPORT.STOCK_CONSUMPTION_GRAPH_REPORT.TITLE',
+      waitFor : 1500,
```

Comment:
What is waitFor?

Reply:
It's useful when the chart as animation, it takes time for being drawn. as I set the animation duration to zero, we can skip this

Reply:
Did I miss something?  Where do you use it?  I can't find any other reference to it in your code.